README.md

```diff
@@ -33,4 +37,32 @@ TBD
 
 ## Documentation
 
-Architectural Decision Records are stored in `doc/adr`
+Architectural Decision Records (ADR) are stored in `doc/adr`
+To create a new ADR, first install [ADR-tools](https://github.com/npryce/adr-tools) if you don't
+already have it installed.
+* `brew install adr`
+
+Then create the ADR:
+*  `adr new Title Of Architectural Decision`
+
+This will create a new, numbered ADR in the `doc/adr` directory.
+
+## Contributing
+
+~This will continue to evolve as the project moves forward.~
+
+* Pull down the most recent main before checking out a branch
+* Write your code
+* If a big architectural decision was made, add an ADR
+* Submit a PR
+  * If you added functionality, please add tests.
+  * All tests must pass!
+* Ping the other engineers for a review.
+* At least one approving review is required for merge.
+* Rebase against main before merge to ensure your code is up-to-date!
+* Merge after review.
+  * Squash commits into meaningful chunks of work and ensure that your commit messages convey meaning.
```

Comment:
I haven't been a big commit squasher throughout my life, but I would have thought that squashing commits before review would help the reviewer more than squashing them after.  Can you explain a bit about your reasoning for putting that in the merge / review step?

Reply:
it's also nice if they're squashed before the review (especially for big reviews, though hopefully we will keep our work small and manageable anyways) but squashing them into meaningful commits and naming it well makes it easier to track what work happened when and why. it's a form of documentation that i value pretty highly, since i do a lot of "what happened here" kinds of code investigations.